docs/integration/kubernetes.md

```diff
@@ -151,23 +153,51 @@ Container and Kubernetes metrics are collected along with labels and tags, which
 
 
 
+*   Read bytes - Is the number of bytes read from the disk
+*   Read time - Represents the total amount of time (in nanoseconds) between read request dispatch and request completion
+*   Read wait time - Represents total amount of time the read I/O operations for the container spent waiting in the scheduler queues
+*   Write bytes - Is the number of bytes written to the disk
+*   Write time - Represents the total amount of time (in nanoseconds) between write request dispatch and request completion
+*   Write wait - time	Represents total amount of time the write I/O operations for the container spent waiting in the scheduler queues
+
+
 ![alt_text](https://sematext.com/wp-content/uploads/2020/04/image1.png "Sematext Kubernetes Metrics")
 
 
 
-### Custom metrics
+### Network
+
+
+*   Received bytes - Received amount of bytes on the network interface
+*   Received packets - Received amount of packets on the network interface
+*   Received errors - Received amount of errors on the network interface
+*   Dropped ingresspackets - The amount of dropped inbound packets on the network interface
+*   Transmitted bytes - Transmitted amount of bytes on the network interface
+*   Transmitted packets - Transmitted amount of packets on the network interface
+*   Transmitted errors - Transmitted amount of errors on the network interface
+*   Dropped egress packets - The amount of dropped outbound packets on the network interface
+
+![alt_text](https://sematext.com/wp-content/uploads/2020/04/network.png "Sematext Kubernetes Metrics")
+
+### Memory
+
+
+*   Memory fail counter - Is the number of times that memory cgroup limit was exceeded
+*   Memory limit - Designates the max allowed memory limit for the container cgroup
+*   Memory pages in - The number of events each time the page is accounted to the container cgroup
+*   Memory pages out - Is the number of events each time a page is unaccounted from the container cgroup
```

Comment:
Why do some lines use "Is the foo..." style and some "The foo..." style?  I see that in a few places.